### PR TITLE
feat: Update releases page to `@primer/css`

### PR DESCRIPTION
**Pull request by HashimotoYT**
| Before | After |
| --- | --- |
| electronjs org_releases_stable | electron-website-pr-2423 herokuapp com_releases_stable |

### CODE REVIEW

**Comment by BinaryMuse** [comment]
I feel like this page is quite wide and leaves a lot of empty space in the center of the screen. Thoughts?

image

**Comment by HashimotoYT** [comment]
Container changed because when viewing very long patches, the page looks small scall (strange word, I'm unable to translate this clearer). I'm fine to change this back to `xl`, and relook that on the redesign in other time.

File: public/styles/_releases.scss [hunk 1]
```diff
@@ -13,75 +17,87 @@
...
+    @include breakpoint(md) { font-size: $h2-size !important; }
+  }
+
+  h4 {
+    font-size: $h3-size-mobile !important;
+    font-weight: $font-weight-light !important;
+
+    @include breakpoint(md) { font-size: $h3-size !important; }
+  }
+
```
Comment (erickzhao) [inline public/styles/_releases.scss:100]: Can get rid of all the `!important` if we wrap this entire block in `.markdown-body`.

File: public/styles/_releases.scss [hunk 2]
```diff
@@ -96,7 +112,7 @@
       &:hover {
         text-decoration: none;
         color: $main-color-strong;
-        background-color: $main-bg-color-shade;
+        background-color: $main-bg-color-shade !important;
```
Comment (erickzhao) [inline public/styles/_releases.scss:115]: Unnecessary `!important`

File: public/styles/_releases.scss [hunk 3]
```diff
@@ -117,7 +133,7 @@
 @media (min-width: 700px) {
   .r-resp-header,
   .r-resp-header-toggle {
```
Comment (erickzhao) [inline public/styles/_releases.scss:135]:
```suggestion
  .btn-link.r-resp-header-toggle {
```

Can remove `!important` afterwards

File: public/styles/_releases.scss [hunk 4]
```diff
@@ -117,7 +133,7 @@
 @media (min-width: 700px) {
```
Comment (erickzhao) [inline public/styles/_releases.scss:133]: Can we use Primer breakpoint variables to standardize things?

File: public/styles/_releases.scss [hunk 5]
```diff
@@ -166,6 +182,6 @@
   }
 
   .r-menu {
-    display: none;
+    display: none !important;
```
Comment (erickzhao) [inline public/styles/_releases.scss:185]: Removing this `!important` still works for me

File: public/styles/_releases.scss (hunk 4 above)
Comment (HashimotoYT) [reply public/styles/_releases.scss]: I updated that to primer min-width, but looks like them doesn't have breakpoints for `@media`

File: public/styles/_releases.scss [hunk 6]
```diff
@@ -13,75 +17,87 @@
...
+    margin-bottom: 1em !important;
+    margin-left: 1.5em !important;
+    padding-left: 0;
+    padding-top: 0;
+  }
 }
 
 .releases {
-  width: 1012px;
+  width: 1280px;
```
[the remark this answers is not in the document]
Comment (HashimotoYT) [reply public/styles/_releases.scss]: This now removed and we only set the margin to `7.5%` for left and right.

**Comment by HashimotoYT** [comment]
Well, my final word it's just to close that since I'm really idk what I should do with that now, because it's now starts to be harder.

**Pull request closed without merging**